### ISSUE TITLE
v1.0.3 - Wizard - When Main menu opens, error "Action failed with message: Invalid capability: text" pops up

### ISSUE THREAD

**Issue by PeterKawa** [Baldhor/Homey-ESPhome-Enhanced#48]
**Describe the bug**
1. After starting the wizard when the Main menu appears, this pop up appears: "Action failed with message: Invalid capability: text". I can hit "OK" and nothing seems to be affected.

**To Reproduce**
1. start the 'ESPhome wizard'
2. when Main menu is in focus, error pops up

**Expected behavior**
No error popping up ;-)

**Screenshots**
![signal-2023-10-03-163041_002](https://github.com/Baldhor/Homey-ESPhome-Enhanced/assets/74005072/f7026946-2f78-40ca-a1ff-1f97aaad95bc)

**Desktop (please complete the following information):**
 - OS: linux Mint 21.2 Cinnamon v.5.8.4
 - Browser: both chrome & firefox
 - Version Chrome: Version 117.0.5938.132 (Official Build) unknown (64-bit) / FF: 118.0.1 (64-bit)

**Smartphone (please complete the following information):**
 - Device: Galaxy s9+ SM-G965F
 - OS: Android 10
 - Browser: both chrome & firefox; both in mobile as in desktop mode
 - Version chrome 117.0.5938.140 / FF: 117.1.0 (Build #2015973691), 6f27adebba+, GV: 117.0.1-20230912013654, AS: 117.0

**Comment by Baldhor** [comment]
@PeterKawa The capability "text" doesn't exist :)
The closest would be the "custom_text" capability but it's broken, see issue #46

Possible to enable console.re and collect some logs so I can understand ?

**Comment by Baldhor** [comment]
https://github.com/Baldhor/Homey-ESPhome-Enhanced/wiki/ConsoleRe-support

**Comment by Baldhor** [comment]
Received a stack trace:
```
Error: Invalid Capability: text
    at Remote Process
    at HomeyClient.emit (/node_modules/@athombv/homey-apps-sdk-v3/lib/HomeyClient.js:44:23)
    at Object.emit (/node_modules/@athombv/homey-apps-sdk-v3/manager/drivers.js:78:54)
    at Object.emit (/node_modules/@athombv/homey-apps-sdk-v3/lib/Driver.js:126:50)
    at VirtualDevice.addCapability (/node_modules/@athombv/homey-apps-sdk-v3/lib/Device.js:446:25)
    at /app/drivers/esphome-wizard/driver.js:412:35
    at PairSession._triggerEvent (/node_modules/@athombv/homey-apps-sdk-v3/lib/PairSession.js:80:12)
    at ManagerDrivers._onPairSessionEvent (/node_modules/@athombv/homey-apps-sdk-v3/manager/drivers.js:190:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by Baldhor** [comment]
Ok actually, there was a 'text' capability lol
It was supposed to be named 'esphome_text', hence the issue ...

Here is what I gonna do:

1. I will remove from definition the existing 'custom' capabilities!
2. When starting the application, I will remove from the existing virtual devices any 'text' or whatever custom capabilities!
3. Through an enhancement I will add a lot of "well made" custom capabilities!

**Tabula Rasa**

**Comment by Baldhor** [comment]
@PeterKawa I published version 1.05 to solved the issue.
I removed completely the text capability support, and at application restart it will removing it from existing virtual devices.

Yes, it's not really a solution, more likely a workaround, but I prefer to redo the custom capabilities from ground up (with icons, and so on) => refer to #41.

Can you confirm it fix your problems?

**Comment by PeterKawa** [comment]
Confirmed. It's fixed with v 1.0.5. (already mentioned via Homey forum). Thanks!

**Comment by PeterKawa** [comment]
Thankssssss! I believe I acked it's working @ forum topic.
(Sorry I'm just not used to git issues and its updates, wasn't ignoring you
🤭 )

Op wo 4 okt. 2023 21:02 schreef Baldhor ***@***.***>:
